Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ccl/configuration/validation/CCLConfigurationJsonSchema.kt

```diff
@@ -0,0 +1,14 @@
+package de.rki.coronawarnapp.ccl.configuration.validation
+
+import android.content.res.AssetManager
+import de.rki.coronawarnapp.util.serialization.validation.JsonSchemaSource
+import javax.inject.Inject
+
+class CCLConfigurationJsonSchema @Inject constructor(
+    private val assetManager: AssetManager
+) : JsonSchemaSource {
+    override val rawSchema: String
+        get() = TODO("Not yet implemented")
+    override val version: JsonSchemaSource.Version
+        get() = TODO("Not yet implemented")
+}
```

Comment:
is it required ? or can we rely on the server side ?

Reply:
I aligned with the Architect about that..
Validation on client side can be removed.
Removed.